sql-plugin/src/main/scala/com/nvidia/spark/rapids/AlluxioUtils.scala

```diff
@@ -578,9 +578,6 @@ object AlluxioUtils extends Logging {
       // we know then fall back to inferring. The latter happens on certain CSPs
       // like Databricks where they have customer file index types.
       relation.location match {
```

Comment:
Overall I did this for performance to specifically get rid of the listleaf files call.   I guess now that we have the Task time replacement algorithm, I would be ok with just removing this code and having ti do the list leaf files all the time to fix this issue and then allow locality to apply.  So please go ahead and remove both case statements and cleanup unneeded code.

Reply:
Updated the case statements.